packages/rpc-parsed-types/src/common.ts

```diff
@@ -0,0 +1 @@
+export type Slot = bigint;
```

Comment:
`Slot` should already be in our other `rpc-types`, no?

Reply:
Sadly, it currently lives in `rpc-core`. That's one of the suggested change I mentioned following the dependency graph.

Reply:
It's in `rpc-core` but it's also not a branded type, just an alias. So I just copied it here instead of refactoring all its uses

No issue with doing that (and rpc-types is the best place I agree), I just didn't think it was important enough to add to this PR

Reply:
+1 on "not a concern for this PR".

Would there be any benefit in making this a branded type? Or is there not much point adding validation to this `bigint` since any `bigint` could in theory be a valid `Slot`?

Reply:
I think it would be worth adding for `Slot` actually because it's used as an input type to RPC methods (`minContextSlot` comes to mind, there are probably others). Not critical but I think it'd add value. 

Probably not worth writing an assertion, we could bound check against u64 I guess but I think that's pretty low value

But having type safety from RPC method returning a slot -> RPC method taking a slot as input is probably worthwhile. Or having to explicitly cast a bigint as a `Slot` to use it as one

Reply:
I've moved this alias to `rpc-types` and imported it from there in `rpc-parsed-types`, just to reduce the future work.